### DISCUSSION TITLE
Satellite view using Google Maps Elementor widget.

### DISCUSSION THREAD

**Discussion by Anthony1070** [elementor/elementor#23097] [Developers Community]
Hello,

I would like to know if it's possible to set the Google Maps widget in order to show the Satellite View of a specific location.

So far I have not been able to find a specific option.

I also thought it may be possible to rach the same result using the html widget with proper coding, but I'm not a coder.

Please let me know how to solve this.

Many thanks in advance.

Cheers.

A.

**Comment by susanhope** [comment]
Hello @Anthony1070 we only just started building this small community, not many people know about it YET! To help you out, we will be sharing a post which will include a link to this post in our main (huge) community on Facebook.
